tests/test_workflows.py

```diff
@@ -119,4 +119,39 @@ async def event_workflow(c: Context):
         assert s.w_output == "Hello from outside! B A"
 
 
-# TODO: test concurrent
+current_time = None
+rand_value = None
+unique_id = None
+
+
+@async_test
+async def test_utils_methods(t: WorkflowTest):
+    def is_durable(a: any, b: any):
+        if a != b:
+            raise Exception(f"not durable: {a} != {b}")
+        pass
+
+    @workflow()
+    async def utils_workflow(context: Context):
+        global current_time, rand_value, unique_id
```

Comment:
I would avoid the behavior of the global and instead use a purest fixture to store the state